grails-app/views/order/show.gsp

```diff
@@ -59,14 +59,16 @@
                                         <format:metadata obj="${orderInstance?.orderTypeCode}"/>
                                     </td>
                                 </tr>
-                                <tr class="prop">
-                                    <td valign="top" class="name">
-                                        <label><warehouse:message code="order.originCode.label"/></label>
-                                    </td>
-                                    <td valign="top" class="value">
-                                        ${orderInstance?.origin?.organization?.code }
-                                    </td>
-                                </tr>
+                                <g:if test="${orderInstance.orderTypeCode == OrderTypeCode.PURCHASE_ORDER}">
```

Comment:
I think this is a second PR, that will need a revisit after merging it to develop. Perhaps we should create a Jira ticket with a list of all PRs that will need refactor into OrderType for easy tracking?